### PR TITLE
conversion rate be a (bound) rational number

### PR DESCRIPTION
**Pull request by deepesh-kn**
This includes the following issue: 
https://github.com/OpenSTFoundation/openst-protocol/issues/107

### CODE REVIEW

File: contracts/BrandedToken.sol [hunk 1]
```diff
@@ -53,7 +53,7 @@ contract BrandedToken is EIP20Token, UtilityTokenAbstract {
         uint256 _chainIdValue,
         uint256 _chainIdUtility,
         uint256 _conversionRate,
-        uint256 _conversionRateDecimalFactor)
+        uint8 _conversionRateDecimalFactor)
```
Comment (benjaminbollen) [inline contracts/BrandedToken.sol]: now it's not the factor anymore and perhaps can be renamed `_conversionRateDecimals`

File: contracts/STPrimeConfig.sol [hunk 2]
```diff
@@ -29,6 +29,7 @@ contract STPrimeConfig {
     string  public constant STPRIME_NAME            = "SimpleTokenPrime";
     uint256 public constant STPRIME_CONVERSION_RATE = 1;
     uint8   public constant TOKEN_DECIMALS          = 18;
+    uint8   public constant STPRIME_CONVERSION_RATE_TOKEN_DECIMALS = 5;
```
Comment (benjaminbollen) [inline contracts/STPrimeConfig.sol]: this constant can be 1

Comment (benjaminbollen) [reply contracts/STPrimeConfig.sol]: correction: it can be 0

File: contracts/BrandedToken.sol (hunk 1 above)
Comment (deepesh-kn) [reply contracts/BrandedToken.sol]: Done

File: contracts/STPrimeConfig.sol (hunk 2 above)
Comment (deepesh-kn) [reply contracts/STPrimeConfig.sol]: Done, changed to 0.

**Pull request merged**